tsconfig.base.json

```diff
@@ -1974,11 +1974,8 @@
     "useUnknownInCatchVariables": false,
     // enabled to point out not supported features when emitting in an isolated module environment
     "isolatedModules": true,
-    // TODO(jbudz): should be uncommented when upgrading to TS@4.8
-    //
     // ensures every non-declaration file is treated as a module
-    // "moduleDetection": "force",
-    //
+    "moduleDetection": "force",
```

Comment:
There's a few seemingly related todo comments introduced in the original PR.  Maybe they can be removed too?

https://github.com/elastic/kibana/pull/144841

Reply:
Good call! Done in [`665158d` (#191834)](https://github.com/elastic/kibana/pull/191834/commits/665158df8ec185459e40d5961e966543a1e57b32)